### ISSUE TITLE
Android - Doesn't return any payment value

### ISSUE THREAD

**Issue by fravezzimattia** [AlejandroRuiz/PayPal.Forms#4]
```
try
{
   var result = await CrossPayPalManager.Current.Buy(new PayPalItem("Carrello Numero " + CarrelloCheckout.id, TotaleCarrello, "EUR"), OrdineCheckout.total_shipping);
  /*MY CODE*/
} catch (Exception e)
{
   Debug.WriteLine(e.Message);
}
```

On Android the payment was send to paypal correctly, but "CrossPayPalManager" Method doesn't return any result!

Thanks

**Comment by AlejandroRuiz** [comment]
Trying to figure out this issue by referencing the project code it works, but isn't work on nuget package

**Comment by fravezzimattia** [comment]
Have you got any news about the bug?

**Comment by fcogutierrez** [comment]
Hi Alejandro.

Could you fix this? Is there any workaround for do this work on Android?

Having the same issue :(

**Comment by AlejandroRuiz** [comment]
Hey @fravezzimattia & @fcogutierrez sorry for the late fix this is fixed now in version 2.0.6 which is now live at nuget
